### DISCUSSION TITLE
Your Azure skills are under validation Its cant take the urls.

### DISCUSSION THREAD

**Discussion by Sai-Durgapu** [github/india#1057] [Students]
![Screenshot (25)](https://user-images.githubusercontent.com/118338970/203031878-2e4678a6-6a4b-434d-9b91-9f5722ba5ffc.png)
Please How to solve this issue? @ey-mentor1, @EY-Mentor3, @ey-mentor2 , @EY-Mentor4

**Comment by Abhisheknik** [comment]
same issue !

**Reply by mudragadasai** [reply]
Same issue

**Reply by saleem1998** [reply]
What happened later

**Reply by teja566** [reply]
same issue please help me to solve this problem

**Comment by teja566** [comment]
same issue please help me to solve this problem
